src/display/editor/editor.js

```diff
@@ -1563,7 +1431,10 @@ class AnnotationEditor {
   select() {
     this.makeResizable();
     this.div?.classList.add("selectedEditor");
-    this.addEditToolbar();
+    if (!this.#editToolbar) {
+      this.addEditToolbar().then(() => this.#editToolbar?.show());
```

Comment:
Is there not a *theoretically* risk that the `unselect`-method, defined below, could have been called before this promise resolves?
In that case the edit-toolbar could incorrectly become visible, so we might need a way to prevent that.